capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx

```diff
@@ -1,28 +1,29 @@
 import {
-  Heading, Badge, Button, HStack, Image, VStack,
+  Heading,
+  Badge,
+  Button,
+  HStack,
+  Image,
+  VStack,
 } from '@chakra-ui/react';
-import { useState, useEffect, useMemo } from 'react';
+import { useState } from 'react';
 
 const API_KEY = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
 
 function getPhotoReference(restaurant) {
-  return restaurant.photos[0].photo_reference;
+  return restaurant.photos ? restaurant.photos[0].photo_reference : null;
 }
 
 export default function ExperienceInfo({ restaurants }) {
   const [indexRestaurant, setIndexRestaurant] = useState(0);
   // Get the restaurant viewing now
-  const currentRestaurant = restaurants.length > 0 ? restaurants[indexRestaurant] : null;
-  // const currentRestaurant = useMemo (
-  //   (restaurants.length > 0 )
-  //   ? (() => restaurants[indexRestaurant], [indexRestaurant, restaurants])
-  //   : null
-  // );
+  const currentRestaurant =
+    restaurants.length > 0 ? restaurants[indexRestaurant] : null;
```

Comment:
This is a bug. we're indexing into `indexRestaurant`, which could be any number, but the condition only checks if there's at least one element.

```suggestion
    restaurants.length > indexRestaurant ? restaurants[indexRestaurant] : null;
```

Reply:
Oh, that was actually so that if there is no restaurants, I don't index. I control the index value within my onNext and onPrevious functions.

Reply:
There could be no restaurants if my app is still loading. I will in the future catch that case and display a loading screen